scl/elasticsearch/plugin.conf

```diff
@@ -68,6 +68,14 @@ block destination elasticsearch2(
   concurrent_requests("1")
   skip_cluster_health_check("")
   cluster_url("")
+  ssl_keystore_filepath("")
+  ssl_keystore_password("")
+  ssl_truststore_filepath("")
+  ssl_truststore_password("")
+  ssl_insecure("false")
+  http_auth_type("none")
+  http_auth_type_basic_username("")
+  http_auth_type_basic_password("")
```

Comment:
Good idea, I'll have a look.

I have a question about the file format of key and cert files: should we prefer standard file format like pkcs12 or the Java default trust/keystore format as created by the `keytool` command?